README.md

```diff
@@ -33,7 +36,7 @@ Live site URL to Our Awesome Books
 ### Install
 
```

Comment:
[OPTIONAL] I see that two developers have worked on this project. Kindly add the contact info of the second developer to your README.

Reply:
Thank you I have just added the second contributor